config/sync/core.entity_view_display.node.document.digest_message.yml

```diff
@@ -52,6 +55,16 @@ content:
       template_suggestion:
         template_suggestion: no_wrappers
     region: content
+  field_document_logo:
+    type: image_url
+    weight: 3
+    region: content
+    label: above
```

Comment:
I checked this and it seems to be rendered normally, no label is visible. Still I will change this to `hidden` so that it is consistent with the other digest message styles.